src/components/PeerInfo.js

```diff
@@ -17,6 +18,7 @@ const SummaryCardItem = styled.div`
 `;
 
 const PeerInfo = ({ peers }) => {
+  const { t } = useTranslation();
```

Comment:
I wonder if we could just call the useTranslation hook at a top level util file, then just import t from there. It would cut out this line here on every file that uses `t` 🤔

Reply:
Hooks can only be called inside of a Component, so not sure if that would work. We might be able to create a wrapper component that does this though.